addon/ActivityStreams.js

```diff
@@ -17,7 +17,6 @@ const {Memoizer} = require("addon/Memoizer");
 const {PlacesProvider} = require("addon/PlacesProvider");
 const {SearchProvider} = require("addon/SearchProvider");
 const {ShareProvider} = require("addon/ShareProvider");
-const {TabTracker} = require("addon/TabTracker");
 const {PreviewProvider} = require("addon/PreviewProvider");
 const {RecommendationProvider} = require("addon/RecommendationProvider");
 const {TelemetrySender} = require("addon/TelemetrySender");
```

Comment:
I was taking a look at how `TabTracker` was actually sending data, and it looks like it sends notifications to `TelemetrySender`? Can you file a followup issue linking this or additionally move `TelemetrySender` init/`uninit` to `main.js`?